src/components/Footer/Footer.tsx

```diff
@@ -1,7 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import waving_girl from '../../assets/images/waving_girl.png';
 import style from './Footer.module.scss';
 
 export const Footer: React.FC = () => (
   <footer className={style.footer}>
-    <div>Footer</div>
+    <div className={style.mainContent}>
+      <h1 className={style.logo}>buk klab</h1>
+
+      <section>
+        <h3 className={style.sectionTitle}>navigation</h3>
+        <nav>
```

Comment:
if you add secondary navigation, add `aria-label` to it, so it is obvious for users with screen readers that they are NOT that the beginning again